src/Mod/PartDesign/Gui/TaskHelixParameters.cpp

```diff
@@ -174,7 +174,9 @@ void TaskHelixParameters::showCoordinateAxes()
             ViewProviderCoordinateSystem* vpOrigin;
             vpOrigin = static_cast<ViewProviderCoordinateSystem*>(
                 Gui::Application::Instance->getViewProvider(origin));
-            vpOrigin->setTemporaryVisibility(true, false);
+            Gui::DatumVisibilityFlags flags;
+            flags.set(Gui::DatumVisibility::Axes);
+            vpOrigin->setTemporaryVisibility(flags);
```

Comment:
Can we use enum bitflags (so properly shifted bits) instead of `std::bitset`? `vpOrigin->setTemporaryVisibility(flags)` is also not great in terms of readability as `flags` does not tell much. I know that in that particular case I only need to look one line above, but it won't be the case always and it can be easy to miss. 

In this case I'd much prefer `vpOrigin->setTemporaryVisibility(Gui::DatumVisibility::Axes)`, especially if this would also be combined with more flags `vpOrigin->setTemporaryVisibility(Gui::DatumVisibility::Axes | Gui::DatumVisibility::Planes)` is much easier to process than having to lookup how flag is defined in terms of function calls that may, or may not be right above. 

Given that we in Gui and we are using Qt anyway we can use QFlags: https://doc.qt.io/qt-6/qflags.html that will handle everything.

`std::bitset` seems fine as flag container when the flags are part of structure and are long-lived (so for example in Document etc) but as short-lived argument they are really not well fitted.